### PR TITLE
[risk=no] revert header changes

### PR DESCRIPTION
**Pull request by SrushtiGangireddy**
1. Revert header changes to look like prod.
2. Flipping precope flag.

### CODE REVIEW

**Comment by SrushtiGangireddy** [comment]
image

image

**Review by mhigbyflowers — APPROVED** [review]
👍  looks good

File: public-ui/src/app/shared/services/header-footer.service.ts [hunk 1]
```diff
@@ -129,6 +129,11 @@ export class HeaderFooterService {
             submenu: false
           }
         ]
+      },
+      {
+        title: 'FAQ',
+        url: this.allOfUsUrl + '/frequently-asked-questions/',
+        submenu: false
       }
```
Comment (mhigbyflowers) [inline public-ui/src/app/shared/services/header-footer.service.ts]: image

**Comment by SrushtiGangireddy** [comment]
Thanks

**Pull request merged**